### PR TITLE
[12.0][IMP] hr_attendance_report_theoretical_time: Add department_id in report

### PR DESCRIPTION
**Pull request by victoralmau**
Add `department_id` field in report
example

@Tecnativa TT27900

### CODE REVIEW

File: hr_attendance_report_theoretical_time/reports/hr_attendance_theoretical_time_report.py [hunk 1]
```diff
@@ -66,6 +74,7 @@ def _select_sub1(self):
     def _from_sub1(self):
         return """
             hr_attendance ha
+            LEFT JOIN hr_employee AS he ON ha.id = he.id
```
Comment (pedrobaeza) [inline hr_attendance_report_theoretical_time/reports/hr_attendance_theoretical_time_report.py]: Isn't this already in https://github.com/OCA/hr/pull/961/files#diff-cf0797000a4e7329aafb13b0890ce2b5d09834ded662f2db6f898a580835f3fcR104 ?

Comment (victoralmau) [reply hr_attendance_report_theoretical_time/reports/hr_attendance_theoretical_time_report.py]: It's necessary to work fine sql and view.

Comment (pedrobaeza) [reply hr_attendance_report_theoretical_time/reports/hr_attendance_theoretical_time_report.py]: Then this is under-performant doing 2 times the same join. We shouldn't add this I'm afraid.

**Comment by pedrobaeza** [comment]
OK, I'm now seeing what you wanted to achieve, and I think it's feasible and not very under-performant, so sorry for the noise and it's OK to revert to previous code.

**Review by pedrobaeza — APPROVED** [review]

**Review by marcelsavegnago — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Trying on runbot, how the same employee can be on 2 departments?

Selección_021

**Comment by victoralmau** [comment]
Thanks for detect error in sql, solved now.

**Pull request merged**